src/core/raster/qgsrasterlayerelevationproperties.cpp

```diff
@@ -241,7 +241,12 @@ bool QgsRasterLayerElevationProperties::layerLooksLikeDem( QgsRasterLayer *layer
   static const QStringList sPartialCandidates{ QStringLiteral( "dem" ),
       QStringLiteral( "height" ),
       QStringLiteral( "elev" ),
-      QStringLiteral( "srtm" ) };
+      QStringLiteral( "srtm" ),
+      // French hints
```

Comment:
```suggestion
      // French hints
      QStringLiteral( "mne" ),
```

Reply:
@nicogodet not sure if didn't include this intentionally.

Seems to be used, e.g. http://eductice.ens-lyon.fr/EducTice/recherche/geomatique/veille/imagerie-3d/MNT/index.htm but you're the native speaker :-)

Reply:
MNE = MNS but you are right, both can be used !